packages/eslint-config-ckeditor5/eslint.config.mjs

```diff
@@ -311,7 +317,11 @@ const rulesTypeScript = [
 				objectLiteralTypeAssertions: 'allow-as-parameter'
 			} ],
 
-			'@typescript-eslint/consistent-type-imports': 'error',
+			'@typescript-eslint/consistent-type-imports': [ 'error', {
+				fixStyle: 'inline-type-imports'
+			} ],
+
+			'@typescript-eslint/consistent-type-exports': 'error',
```

Comment:
Let remove this rule for now because it has a bug, which reports file which re-exports `.css` files as **type** export.

Soon we will likely enable `isolatedModules` and `isolatedDeclarations` in TypeScript and the former covers the same thing as this rule ([more information](https://typescript-eslint.io/rules/consistent-type-exports/#when-not-to-use-it)).